### DISCUSSION TITLE
Should an ApplicationSet be responsible for one service only?

### DISCUSSION THREAD

**Discussion by varsis** [argoproj/argo-cd#16052] [Q&A]
Should an ApplicationSet be responsible for one service only? For example I have a domain and it contains multiple services. Should the ApplicationSet have all of the services in a single ApplicationSet or should each individual service have it's own ApplicationSet? 

If they should be individual, is there a way to group them? Similar to App of Apps.

**Comment by FernandoMiguel** [comment]
it's up to you what pattern you pick.
either works
